.github/workflows/cd.yml

```diff
@@ -19,10 +19,15 @@ jobs:
     permissions:
       contents: read
       packages: write
-      # 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+      - name: Set up QEMU
```

Comment:
> QEMU is a generic and open source machine & userspace emulator and virtualizer.
QEMU is capable of emulating a complete machine in software without any need for hardware virtualization support

This is needed to emulate all the OS/platforms